src/importlinter/domain/contract.py

```diff
@@ -43,7 +48,11 @@ def _populate_fields(self) -> None:
             except fields.ValidationError as e:
                 errors[field_name] = str(e)
                 continue
-            setattr(self, field_name, clean_data)
+
```

Comment:
Another approach would be to change `BaseMultipleValueField` to ignore empty lines directly, or change `field.parse(raw_data)` to ignore empty strings. I'm not sure if that would be better, so I decided to add a sentinel value that only affects the `ImportExpressionField` type.

Reply:
I think changing `BaseMultipleValueField` is the better approach. It means we benefit from this change in other fields such as `exhaustive_ignores` too. I also prefer the idea of maintaining strictness at the individual field level. It adds complexity if an `ImportExpressionField` can give you a `DummyValue` - what if you wanted to use this as a single field, rather than a multiple value field?

I'm comfortable with empty lines having no meaning for any multiple value fields, and we just use them for visual separation.